dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:10.16.0-alpine
```

Comment:
@Jumaruba  Why are we not using a more recent version of node?

Reply:
Updated to the lts version in commit 615ee78b928f42c930dde7d2b189e4c007b38fc4